game/static/game/css/landing.css

```
@@ -65,6 +65,18 @@ body {
 .logo-img {
     height: 35px;
     width: auto;
+    animation: logoGlow 2.5s ease-in-out infinite alternate;
+}
+
+@keyframes logoGlow {
```

Stylelint (17.11.0)

[error] Expected keyframe name "logoGlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)